test/cmdlineTests/revert_strings/output

```diff
@@ -0,0 +1,362 @@
+IR:
+/*******************************************************
+ *                       WARNING                       *
+ *  Solidity to Yul compilation is still EXPERIMENTAL  *
+ *       It can result in LOSS OF FUNDS or worse       *
+ *                !USE AT YOUR OWN RISK!               *
+ *******************************************************/
+
+
+object "C_15" {
+    code {
+        mstore(64, 128)
+        if callvalue() { revert_error_ca66f745a3ce8ff40e2ccaf1ad45db7774001b90d25810abd9040049be7bf4bb() }
+
+        constructor_C_15()
+
+        let _1 := allocate_unbounded()
+        codecopy(_1, dataoffset("C_15_deployed"), datasize("C_15_deployed"))
+
+        return(_1, datasize("C_15_deployed"))
+
+        function allocate_unbounded() -> memPtr {
+            memPtr := mload(64)
+        }
+
+        function constructor_C_15() {
+
+        }
+
+        function revert_error_ca66f745a3ce8ff40e2ccaf1ad45db7774001b90d25810abd9040049be7bf4bb() {
+            {
+                let start := allocate_unbounded()
+                let pos := start
```

Comment:
I see that the body includes extra braces. The code is still valid but just pointing it out in case it was not intentional.

Reply:
It was intentional ,but probably not clever. I will change it again.